src/services/ClientService.java

```diff
@@ -0,0 +1,119 @@
+package services;
+
+import models.client.Client;
+import models.events.Event;
+import models.ticket.Ticket;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class ClientService {
+    static ArrayList<Client> clients = new ArrayList<Client>();
+    static Integer id = 0;
+
+    public void createClient(String parameters) {
+        id++;
```

Comment:
[Improvement] acest id static se poate tine si in entitate, si sa se incrementeze de cate ori este apelat constructorul